prac_07/programming_language.py

```diff
@@ -0,0 +1,16 @@
+
+class ProgrammingLanguage:
+
+    def __init__(self, name, typing, reflection,year):
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def __str__(self):
+        return "{}, {} Typing, Reflection={}, First appeared in {}".format(self.name,self.typing,self.reflection,self.year)
+
+    def is_dynamic(self):
+        if self.typing == "Dynamic":
+            return True
```

Comment:
Where's the `else: return False`?
This will return `None` instead of False, which is not the same.